exchangelib/folders.py

```diff
@@ -1561,6 +1561,7 @@ class Message(ItemMixIn):
         'to_recipients': ('ToRecipients', [Mailbox]),
         'cc_recipients': ('CcRecipients', [Mailbox]),
         'bcc_recipients': ('BccRecipients', [Mailbox]),
+        'message_id': ('InternetMessageId', string_type),
```

Comment:
This one was committed separately in https://github.com/ecederstrand/exchangelib/commit/e2290299c65a6f674ec0a1d7cb4bc89010c17441 with the addition of marking it as a read-only field.